src/component/Footer/Footer.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FOOTER_LIST_TOP, FOOTER_LIST_BOTTOM } from './FOOTER_LIST';
+
+import './Footer.scss';
+
+class Footer extends React.Component {
+  render() {
+    const topList = FOOTER_LIST_TOP.map(elem => {
+      return (
+        <li key={elem.id}>
+          <Link to={elem.url}>{elem.text}</Link>
+        </li>
+      );
+    });
```

Comment:
```suggestion
    const topList = FOOTER_LIST_TOP.map(elem => {
     const {id, text} = elem;
      return (
        <li key={id}>
          <Link to={url}>{text}</Link>
        </li>
      );
    });
```
구조분해할당해서 사용하면 깔끔하겠네요